src/components/InventoryGroups/utils/api.js

```diff
@@ -0,0 +1,21 @@
+import { instance } from '@redhat-cloud-services/frontend-components-utilities/interceptors/interceptors';
+import { INVENTORY_API_BASE } from '../../../api';
+
+//had to replace the instance.post with axios.post otherwise I had an error because chrome is missing
+export const createGroup = (payload) => {
+    return instance.post(`${INVENTORY_API_BASE}/groups`, {
+        Name: payload.name,
+        Type: 'static'
+    });
+};
+
+export const addSystemsToGroup = (groupId, systems) => {
+    return instance.post(`${INVENTORY_API_BASE}/groups/${groupId}`, {
+        ID: groupId,
+        Devices: systems
+    });
+};
+
+export const validateGroupName = (name) => {
+    return instance.get(`${INVENTORY_API_BASE}/groups/${name}`);
```

Comment:
There is no such endpoint in the API spec. You can only perform GET /groups/ID with ID, not name.

Reply:
Updated the validate group name function